### PR TITLE
Meeting notes from 22/06/2020

### PR DESCRIPTION
**Pull request by doruchan**

### CODE REVIEW

File: meetings/2020-06-22.md [hunk 1]
```diff
@@ -1,17 +1,100 @@
...
+## Current blockers
+- Ollie: we are blocked by not knowing dates
+- Mark: there are some things that we can do like import future. Are people doing that?
+- Ryan: we should futurizing our code but there's no motivation as we don't have any dates
+- Ashley: it's a big challenge and it's going to take a lot of development time. we want to spread it 
+- Ricky: similar, we want to spread as much as possible 
+- Ashley: how? we have been thinking to use a tool called "python modernise" 
+- Ricky: we are still looking at it with python linters, 2to3 reports
+- Ryan: we use pylint with the -3k flag and capturing what numbers of issues we have in our repos
+- Ashley: we have looked that pylint reports look practically identical. I'm curious to see how many false positives we are going to have
```
Comment (AWhetter) [inline meetings/2020-06-22.md]: To clarify this point, I mentioned that the pylint reports are almost identical to the `python-modernize` ones.

File: meetings/2020-06-22.md [hunk 2]
```diff
@@ -1,17 +1,100 @@
...
+- Ashley: rather than creating a list of links would you suggest writing the docs ourselves?
+- Ryan: no, more like curating the list of links rather than dump everything in 1 page. not sure how we would measure that 
+- Ollie: who is going to start on the resources?
+- Ashley: I am going to start writing
+- Ryan: other big topic: python bindings aspect of it - it can be difficult or not, depending on the setup. it is another big aspect that will hit us fairly hard. 
+- Ashley: last time it was mentioned it was hard to switch with pybind, what are the downsides etc. depending on the code base it takes a very long time to switch and we were working with a fairly significantly sized piece of code. I am afraid for the renders
+- Ryan: it's very difficult without something to link against, i.e. a version of a DCC with a compatible version of python. You don't know it until you can run it
+- Ashley: for pyside, there's a tool called stubgen, for generating function stubs for your pybind code. we are experiment with that atm. we want to try static linting and type checking with that stuff, esp cause our test coverage is not great.
```
Comment (AWhetter) [inline meetings/2020-06-22.md]: Typo: "For pybind" instead of "pyside" ;)

**Review by AWhetter — APPROVED** [review]

File: meetings/2020-06-22.md [hunk 3]
```diff
@@ -1,17 +1,100 @@
...
+- Ollie: is it genuinely that the only blocker is that we don't have a date?
+- Ryan: I would say that there's no blocker - it's end of life and we should do it. having DCC support is important but there's really nothing impeding us to make OUR codebase p3 compatible. I think it's more procrastination. it's an easy excuse to not do it because it can take forever
+- Ashley: a large portion of our code cannot be tested in p3 and code that is DCC specific
+- JT: Paul Hildebrant from Disney mentioned that at Siggraph: https://groups.google.com/forum/#!topic/vfx-platform-discuss/Nfsl8Uf-dpU
+"Here are the text and notes for my slides from the Siggraph VFX Reference Platform BOF."
+- Ricky: one of the things we trying to hammer is the education aspect - so people start writing in p3 and we don't have to re-teach them
+- Ryan: we have found topics that we need to dwell in
+- Ricky: dictionary usings and views - you have to pick your poison
+- Ashley: switch to unicode as default, string no longer the same as bytes it's the most diff thing to do. You don't have to thing about it too much but it's more about using p3 in the future and backport
```
Comment (RoseAllenM) [inline meetings/2020-06-22.md]: Typo:  You don't have to **think** about it..

File: meetings/2020-06-22.md [hunk 4]
```diff
@@ -1,17 +1,100 @@
...
+- Dora: what can we do to help education? i.e. it would be nice to curate a list of tutorials, talks, documentations..
+- Ryan: a good way would be to involve vendors that can provide specific points that have changed significantly - some points might matter more for some DCCs. the python docs is available out there, but the DCC specific doesn't exit so we can make sure it will exist
```
Comment (RoseAllenM) [inline meetings/2020-06-22.md]: Typo:  the DCC specific doesn't **exist**

**Comment by AWhetter** [comment]
Thanks again for taking these minutes @doruchan

**Pull request merged**